web/src/views/Home/parts/DefaultLimits.tsx

```diff
@@ -70,6 +72,10 @@ const DefaultLimits = () => {
       refetch();
       handleModalClose();
     },
+    onError: () =>
```

Comment:
Does this need to be added to the `/info` and `/limits` calls too?

Reply:
Sure, I'll display an error for those too

Reply:
@joostjager Done